incubator/hnc/api/v1alpha1/hnc_config.go

```diff
@@ -46,7 +46,6 @@ const (
 // HNCConfigurationCondition codes. *All* codes must also be documented in the
 // comment to HNCConfigurationCondition.Code.
 const (
-	CritSingletonNameInvalid         HNCConfigurationCode = "CritSingletonNameInvalid"
```

Comment:
Did you remove the documentation as well? See line 46, above.

Reply:
Ah good catch, will update it!